esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -29,9 +28,14 @@ def _get_mesh_and_dim(cube):
     pass
 
 
-def _cube_to_MeshInfo(cube):
-    # Returns a MeshInfo object describing the mesh of the cube.
-    pass
+def _mesh_to_MeshInfo(mesh):
+    assert mesh.topology_dimension == 2
+    meshinfo = MeshInfo(
+        np.stack([coord.points for coord in mesh.node_coords], axis=-1),
```

Comment:
The order in which mesh.node_coords returns its respective coordinates is controled by https://github.com/SciTools/iris/blob/3b8f492e1a16c9cbd50dd3758f96f3178c40c737/lib/iris/experimental/ugrid.py#L2126